azurerm/internal/services/cosmos/common/schema.go

```diff
@@ -89,6 +89,37 @@ func CosmosDbIndexingPolicySchema() *schema.Schema {
 						},
 					},
 				},
+				"composite_index": {
+					Type:     schema.TypeList,
+					Optional: true,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"index": {
+								Type:     schema.TypeList,
+								MinItems: 1,
+								Required: true,
+								Elem: &schema.Resource{
+									Schema: map[string]*schema.Schema{
+										"path": {
+											Type:         schema.TypeString,
+											Required:     true,
+											ValidateFunc: validation.StringIsNotEmpty,
+										},
+										"order": {
+											Type:     schema.TypeString,
+											Required: true,
+											ValidateFunc: validation.StringInSlice(
+												[]string{
+													"ascending",
+													"descending",
+												}, true),
```

Comment:
all new fields are case-sensitive by default - so this wants to be:

```suggestion
												}, false),
```